src/Yoti.Auth/ShareUrl/Policy/DynamicPolicyBuilder.cs

```diff
@@ -111,17 +108,21 @@ public DynamicPolicyBuilder WithSelfieAuthentication(bool enabled)
         {
             if (enabled)
             {
-                return WithAuthType(_selfieAuthType);
+                return WithAuthType(DynamicPolicy.SelfieAuthType);
             }
+
+            _wantedAuthTypes.Remove(DynamicPolicy.SelfieAuthType);
```

Comment:
Should this be in `WithAuthType()` with new enabled argument, i.e. `WithAuthType(int authType, bool enabled)`, or are we only allowing selfie and pin to be removed?

Reply:
Good point, I originally had `WithoutAuthType`, but thought that sounded silly! I think `WithAuthType(int authType, bool enabled)` seems sensible, and fitting with the rest of how we build dynamic policies.

Reply:
Added in 4be0e64